packages/contracts-bedrock/test/libraries/StaticConfig.t.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { Test } from "forge-std/Test.sol";
+import { FFIInterface } from "test/setup/FFIInterface.sol";
+
+// Target contract
+import { StaticConfig } from "src/libraries/StaticConfig.sol";
+
+contract StaticConfig_Test is Test {
+    FFIInterface constant ffi = FFIInterface(address(uint160(uint256(keccak256(abi.encode("optimism.ffi"))))));
+
```

Comment:
Something small that would be nice but not required (especially not as part of this PR) would be to modularize setting `ffi` into its own small contract to dedupe this pattern across the codebase